base/error.go

```diff
@@ -39,6 +39,7 @@ var (
 	ErrIndexAlreadyExists    = &sgError{"Index already exists"}
 	ErrNotFound              = &sgError{"Not Found"}
 	ErrUpdateCancel          = &sgError{"Cancel update"}
+	ErrImportIgnored         = &sgError{"Ignore Import"}
```

Comment:
I feel like we could make this more precise until we actually have another use case for it - calling it ErrImportCancelledPurged would make it crystal clear why it's being ignored.